xentoollog.opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Mock OCaml bindings for Xen's xentoollog"
+maintainer: "lindig@gmail.com"
+authors: "lindig@gmail.com"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/xapi-project/xenctrl"
+bug-reports: "https://github.com/xapi-project/xenctrl/issues"
+depends: [
+  "ocaml"
+  "dune" {build}
+  "base-unix"
+]
+build: ["dune" "build" "-p" name "-j" jobs]
+depexts: [
+  ["m4" "libxen-dev" "libsystemd-dev"] {os-distribution = "debian"}
+  ["libxen-dev" "uuid-dev"] {os-distribution = "ubuntu"}
```

Comment:
do we really need these? I thought we just expose a mock environment here that doesn't use the real Xen. Can we drop the depexts?

Reply:
I think it still needs the headers to compile the stubs:

```h
#include <xentoollog.h>

#include "caml_xentoollog.h"
```
https://github.com/xapi-project/xenctrl/blob/master/libs/xentoollog/xentoollog_stubs.c#L30-L32

AFAIK these are stable bindings, so updating these shouldn't break LTS releases.

On top of that all distributions should have easy access to xen headers

Reply:
I missed these files are included here, changing

Reply:
I'll check the other libraries as xenctrl still defines depexts